ios/demo/Info.plist

```diff
@@ -43,6 +39,10 @@
 	</dict>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string></string>
+	<key>UIAppFonts</key>
+	<array>
+		<string>Entypo.ttf</string>
```

Comment:
Why don't you apply the same fonts that used in the design?

Reply:
I looking to add the theme to this project after finishing add the authentication flow. so I'm not sure that I need to set the font in this file or in the theme file

Reply:
Please let me know when you sure.